packages/@uppy/google-drive/src/DriveProviderViews.js

```diff
@@ -0,0 +1,18 @@
+const ProviderViews = require('@uppy/provider-views')
+
+module.exports = class DriveProviderViews extends ProviderViews {
+  constructor (plugin, opts) {
+    super(plugin, opts)
+    this.originalToggleCheckbox = this.toggleCheckbox
+    this.toggleCheckbox = this.toggleDriveCheckbox.bind(this)
+  }
+  toggleDriveCheckbox (e, file) {
```

Comment:
Is there a reason not to name this `toggleCheckbox` and then do `super.toggleCheckbox(e, file)` below? I don't _think_ the .bind() stuff should interfere but i may be wrong…

Reply:
I did try several approaches to this, but this was the only one which let me call the base `ProviderView`'s `toggleCheckbox`.  I mustn't have tried `super.toggleCheckbox()` though, as that does indeed seem to work.  That does make it a bit cleaner - thanks!